### DISCUSSION TITLE
怎么实现x 和 y的线

### DISCUSSION THREAD

**Discussion by zhiyou1201** [antvis/G2#5728] [Q&A]
怎么实现 y 有线 和 x 有线 像这样的 而且怎么实现 光标在空白处也可以让柱状图显示出来信息， 咱们的现在是鼠标必须在柱状图上面才显示
![image](https://github.com/antvis/G2/assets/75651528/f6cd45f5-5032-453c-ab76-1e2e8621708c)

https://github.com/antvis/G2/assets/75651528/9ee92829-2eb6-4bcb-815a-5e6f646c264b

**Comment by pearmini** [comment]
目前还不支持这个功能，要实只能自己自定义交互。

**Reply by zhiyou1201** [reply]
自定义交互怎么写

**Reply by pearmini** [reply]
参考这个：https://observablehq.com/@pearmini/draggable-pie-chart-for-g2
